rconweb/api/history.py

```diff
@@ -74,6 +74,34 @@ def get_player(request):
     )
 
 
+@csrf_exempt
+@login_required
+def get_player_ban(request):
```

Comment:
You don't need that endpoint anymore. every method of the Extended_commands class is automatically turned into a endpoint.

So now that you added a `get_ban` method, you have an enpoints for free at `/api/get_ban?steam_id_64=....`

Reply:
Didn’t see that endpoint